apps/frontend/src/libs/modules/api/base-http-api.ts

```diff
@@ -110,6 +110,10 @@ class BaseHTTPApi implements HTTPApi {
 
 		const headers = await this.getHeaders(contentType, hasAuth);
 
+		if (payload instanceof FormData) {
+			headers.delete(HTTPHeader.CONTENT_TYPE);
+		}
```

Comment:
Why not add multipart/form-data?

Reply:
If you remove it later, why do you add it in the first place?

Reply:
Do you mean why I added it to the ContentType but then deleted it before sending the request? Then I can remove this field in this enum, and pass ContentType.JSON to the request, then this header will be removed and automatically set multipart/form-data when requested

Reply:
I can't understand, why do you want to pass application/json? In getHeaders add a check and don't add the header, if it is multipart